guardian/media-atom-maker
# Update Node version

Co-authored-by: @AWare 

## What does this change?
1. sets Node to version 14 in the app
2. sets `pluto-message-ingester` Node to version 14 and reinstates pluto .nvmrc (deleted in https://github.com/guardian/media-atom-maker/pull/995)
3. splits out the CI steps to run in Docker.

## How to test
Deploy build 4352 to CODE and test Pluto messages. This was tested with @fredex42 and works as expected.
